cmd/kaniko-acr/main.go

```diff
@@ -368,7 +381,11 @@ func fetchACRToken(tenantId, token, registry string) (string, error) {
 }
 
 func setupACRCert(cert string) error {
-	err := ioutil.WriteFile(ACRCertPath, []byte(cert), 0644)
+	decoded, err := base64.StdEncoding.DecodeString(cert)
+	if err != nil {
+		return err
```

Comment:
return errors.Wrap(err, "failed to base64 decode ACR certificate")

Reply:
done